apiserver/facades/client/application/application.go

```diff
@@ -235,25 +229,15 @@ type DeployApplicationFunc = func(
 // NewAPIBase returns a new application API facade.
 func NewAPIBase(
 	backend Backend,
-	ecService ExternalControllerService,
-	networkService NetworkService,
+	services Services,
```

Comment:
Love it, we should start passing the whole `Services` when we have facades taking more than 2 or 3 services.